becquerel/tools/materials.py

```diff
@@ -81,7 +81,7 @@ def _load_and_compile_materials():
             "formula": formula,
             "density": density,
             "weight_fractions": weight_fracs,
-            "source": '"NIST (http://physics.nist.gov/PhysRefData/XrayMassCoef/tab1.html)"',  # noqa: E501
+            "source": '"NIST (http://physics.nist.gov/PhysRefData/XrayMassCoef/tab1.html)"',
```

Comment:
why two sets of quotes?

Reply:
Here and lines 98 and 116. It looks like original to the code I wrote, and yes it looks like it's not necessary.